modular_skyrat/modules/customization/modules/culture/culture/_culture.dm

```diff
@@ -1 +0,0 @@
-/datum/cultural_info/culture
```

Comment:
lmao

Reply:
Some of the lore entries are gold, it's hilarious